src/server/pfs/cmds/cmds.go

```diff
@@ -458,43 +462,26 @@ $ {{alias}} foo@master --from XXX`,
 	listCommit.Flags().StringVarP(&from, "from", "f", "", "list all commits since this commit")
 	listCommit.Flags().IntVarP(&number, "number", "n", 0, "list only this many commits; if set to zero, list all commits")
 	listCommit.MarkFlagCustom("from", "__pachctl_get_commit $(__parse_repo ${nouns[0]})")
-	listCommit.Flags().AddFlagSet(rawFlags)
-	listCommit.Flags().AddFlagSet(fullTimestampsFlags)
+	listCommit.Flags().AddFlagSet(outputFlags)
+	listCommit.Flags().AddFlagSet(timestampFlags)
 	shell.RegisterCompletionFunc(listCommit, shell.RepoCompletion)
 	commands = append(commands, cmdutil.CreateAlias(listCommit, "list commit"))
 
 	var branches cmdutil.RepeatedStringArg
 	waitCommit := &cobra.Command{
-		Use:   "{{alias}} ( <commitset-id> | <repo>@<branch-or-commit> )",
-		Short: "Wait for the specified commit(s) to finish and return them.",
-		Long:  "Wait for the specified commit(s) to finish and return them.",
+		Use:   "{{alias}} <repo>@<branch-or-commit>",
+		Short: "Wait for the specified commit to finish and return it.",
+		Long:  "Wait for the specified commit to finish and return it.",
 		Example: `
```

Comment:
move these examples down to wait commitset

Reply:
ah, good catch